runtime/commands/fun.js

```diff
@@ -62,12 +62,16 @@ Commands.fortunecow = {
   timeout: 20,
   level: 0,
   fn: function (msg) {
-    unirest.get('https://thibaultcha-fortunecow-v1.p.mashape.com/random')
```

Comment:
Does this API still work? Last time I've checked it was down.
If it still appears down, could you remove this command instead?